doc/cephfs/mount-using-fuse.rst

```diff
@@ -55,6 +55,16 @@ If you have more than one FS on your Ceph cluster, use the option
 
 You may also add a ``client_fs`` setting to your ``ceph.conf``
 
+You may specify a mapping between the local user's UID (or GID) and the remote
+server's UID (or GID) by using::
+
+    ceph-fuse --id foo --uid-mapping uid_map_file_name --gid-mapping gid_map_file_name
```

Comment:
I don't think that's right @vshankar. The utility of something like this is letting a client machine do arbitrary local mappings, as things change. We don't want to encoded that into something that the servers have to maintain and account for.

Reply:
I interpret "things change" as a frequent change in id-mapping that needs to be performed. Would that be correct? What's interesting is the comment here https://github.com/ceph/ceph/pull/45667#issuecomment-1088599380 sounds otherwise.

Anyway, does the mapping need to read from a file? Does the client need to maintain all mappings or would a single mapping (per mount) suffice?

Reply:
ping @zv0n @gregsfortytwo